root/layout/index.js

```diff
@@ -88,6 +89,30 @@ const Layout = ({$c, ...props}) => (
           </p>
         </div>}
 
+      {$c.user &&
+      $c.user.birth_date &&
+      moment($c.user.birth_date).month() === moment().month() &&
+      moment($c.user.birth_date).date() === moment().date() &&
+      !getRequestCookie($c.req, 'birthday_message_dismissed_mtime') &&
+      <div className="banner birthday-message">
+        <p>
+          <span aria-label="cakes" role="img">
```

Comment:
Cool! But if the label is going to be read by screen readers, we should probably translate that and be slightly more descriptive, right?
```suggestion
          <span aria-label={l('Birthday cakes')} role="img">
```

Reply:
Did that, and moved it into a separate component instead of repeating it twice.